up-l1/zenoh.adoc

```diff
@@ -253,7 +240,8 @@ The way to transfer Response message is different from others, because Zenoh can
 ==== Message routing
 
 1. A streamer that is interested in all incoming messages for dest-authority registers a listener for
-    * `up://dest-authority////`
+    * source: `-`
+    * sink: `up://dest-authority/FFFF/FF/FFFF`
```

Comment:
@evshary -- I'm honestly not sure what's better from a readability standpoint. I think we could also represent this as:
```
* sink: `up://dest-authority/0000FFFF/FF/FFFF`
```
according to the [spec](https://github.com/eclipse-uprotocol/up-spec/blob/main/basics/uri.adoc#611-matching-on-authority), right?

Reply:
I agree with you that this is more readable.
The reason why I omit the `0000` is that the example in spec always does so.
Since the entity includes service and instance, it might be better to show the whole hex string in entity and avoid the ambiguity.

Reply:
After a second look, it might be better for the designer to give us some direction.
@sophokles73 Do you think we should use `0000FFFF` instead of `FFFF` to match all any instances and any types in ue_id?

BTW, out of curiosity, why do you choose `0000` to stand for any instance?

Reply:
> BTW, out of curiosity, why do you choose 0000 to stand for any instance?

In almost all cases, the sender of a message does not care about which service instance its request is being sent to. Using `0x0000` for the 16 MSB in this case will lead to a URI that is four characters shorter than including `0xFFFF`, because we can simply omit the leading zeros.

Reply:
> Do you think we should use 0000FFFF instead of FFFF to match all any instances and any types in ue_id?

It doesn't make any difference FMPOV